dotnet/Capstone/DAO/BirdListSqlDao.cs

```diff
@@ -0,0 +1,212 @@
+﻿using Capstone.Exceptions;
+using Capstone.Models;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data.SqlClient;
+
+namespace Capstone.DAO
+{
+    public class BirdListSqlDao : IBirdListDao
+    {
+        private readonly string connectionString;
+        private readonly IUserDao userDao;
+
+
+        public BirdListSqlDao(string dbConnectionString)
+        {
+            connectionString = dbConnectionString;
+            this.userDao = new UserSqlDao(connectionString);
+
+        }
+
+        public BirdList createList(BirdList list, string username)
+        {
+            BirdList newBirdList = null;
+
+            string sql = "INSERT INTO list(user_id, name) VALUES((SELECT user_id FROM users WHERE username = @username), @name )";
+
+            int newListId = 0;
+
+            try
+            {
+                using (SqlConnection conn = new SqlConnection(connectionString))
+                {
+                    conn.Open();
+                    SqlCommand cmd = new SqlCommand(sql, conn);
+                    cmd.Parameters.AddWithValue("@username", username);
+                    cmd.Parameters.AddWithValue("@name", list.ListName);
+
+                    newListId = Convert.ToInt32(cmd.ExecuteScalar());
+                }
+
+                newBirdList = getList(newListId);
+            }
+            catch(SqlException ex)
+            {
+                throw new DaoException("SQL exception error", ex);
+            }
+            return newBirdList;
+        }
+
+        public void deleteList(int listId)
```

Comment:
This method could probably use a special handling case to check if there are any birds in the list before deleting, or an Exception that is thrown in that outcome.

Reply:
What will be the use of such handling given that the method is deleting the list whether it has birds or not.